docs/manual/developer/04_style_guide.md

```diff
@@ -345,9 +345,31 @@ Value must be low, medium, or high.
 ### Ansible
 
 * Shall follow all the rules in the [YAML](#yaml) section
-* Should prefer using Ansible modules over just calling system commands
-* Shall be written to pass [`ansible-lint`](https://github.com/ansible-community/ansible-lint)
-* Task names should be prefixed by `{{{ rule_title }}}`, e.g. `- name: "{{{ rule_title }}} - ensure correct banner"`
+* Shall use fully-qualified collection names [(FQCN)](https://ansible-lint.readthedocs.io/rules/fqcn/). e.g. use `ansible.builtin.lineinfile:` instead of only `lineinfile:`
+* Shall use specific Ansible modules whenever possible instead of just calling system commands with `command`, `shell` or `raw` modules
+    * When no specific Ansible module is available, `command` module should be preferred instead of `shell` or `raw` modules
```

Comment:
```suggestion
    * When no specific Ansible module is available, `command` module shall be used instead of `shell` or `raw` modules when the  `command` module is sufficient.
```
We really shouldn't be using `shell` or `raw` unless we really need to. Please feel free adjust my wording.

Reply:
Ok. I will adopt the suggestion and remove the extra space before `command`. : )